### PR TITLE
add: requestAndConfirmAirdrop()

### PR DESCRIPTION
**Pull request by mikemaccana**
So [the new web3.js has requestAndConfirmAirdrop()](https://github.com/solana-labs/solana-web3.js/blob/b31535247346c200cbc728cf4c108b2bbcfb1d00/packages/library/src/airdrop.ts#L30). Which is great, but the new web3.js is not out yet. 

We need a quick way to have airdrops 'just work' in the meantime. 

@nickfrosty this isn't technically node. Maybe we should make a `browser-helpers` and add it there instead?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,20 @@ await addKeypairToEnvFile(testKeypair, "SECRET_KEY", ".env.local");
 
 This will also reload the env file
 
+## requestAndConfirmAirdrop()
+
+Request and confirm an airdrop in one step. This is built the next version of web3.js, but we've added it here now for your convenience.
```
Comment (nickfrosty) [inline README.md:61]:
```suggestion
Request and confirm an airdrop in one step. This is built into the next version of web3.js, but we've added it here now for your convenience.
```

File: src/index.ts [hunk 2]
```diff
@@ -92,3 +92,21 @@ export const addKeypairToEnvFile = async (
...
+  publicKey: PublicKey,
+  amount: number,
+) => {
+  let airdropTransactionSignature = await connection.requestAirdrop(
+    publicKey,
+    amount,
+  );
+  // Wait for airdrop confirmation
+  const latestBlockHash = await connection.getLatestBlockhash();
+  await connection.confirmTransaction({
```
Comment (nickfrosty) [inline src/index.ts:107]: I think we should return the promise
```suggestion
  return await connection.confirmTransaction({
```

**Review by nickfrosty — CHANGES_REQUESTED** [review]
I think adding this helper into here is a great idea

File: src/index.ts (hunk 2 above)
Comment (mikemaccana) [reply src/index.ts:107]: Excellent idea, people might want to check aspects of the confirmed transaction!

**Review by nickfrosty — APPROVED** [review]

**Comment by mikemaccana** [comment]
Hrm it doesn't seem to work unless we set a finalisation strategy on connection, even when explicitly setting a commitment level on `connection.confirmTransaction()`. Investigating, may ask for help.

**Comment by mikemaccana** [comment]
Rebased, fixed some bugs, made it return the new balance (which seems the most handy thing) and updated the docs.

File: .github/workflows/tests.yaml [hunk 3]
```diff
@@ -24,7 +24,10 @@ jobs:
       - name: Add to PATH
         shell: bash
         run: |
-          echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+          echo "~/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
```
Comment (mikemaccana) [inline .github/workflows/tests.yaml:27]: I've already merged in `getCustomErrorMessage()` so once #9 is merged this PR will look a lot smaller.

**Pull request merged**